Test Automation Course/src/abstraction/FormaGeometrica.java

```diff
@@ -0,0 +1,6 @@
+package abstraction;
+
+public abstract class FormaGeometrica {
+
+    public abstract void calculArie();
```

Comment:
ar fi mai practic sa schimbi return type in float, si  cind apelezi aceasta metoda deodata sa returnezi aria calculata in implementari

Reply:
Done